dc-model-jackson/src/test/resources/serializedTestObjects/identifiable/alias/LocalizedUrlAliases.json

```diff
@@ -13,6 +13,7 @@
       "uuid" : "9b676195-f505-400c-acb2-b7aabb7c62f7",
       "identifiableObjectType" : "WEBSITE",
       "identifiers" : [ ],
+      "subjects" : [ ],
```

Comment:
Hätte es eine Möglichkeit gegeben, dass eine leere Liste an Subjects bei der JSON-Serialisierung unterdrückt wird?

Reply:
Das wäre jetzt gelöst. Dabei fallen auch eine Reihe anderer leerer JSON-Arrays weg. Folglich müssen auch die Controllertests in cudami angepasst werden.